### ISSUE TITLE
Questions about LCM model training

### ISSUE THREAD

**Issue by luochunfen** [prs-eth/Marigold#103]
Hi! I'm trying to train the LCM model. However, the model I trained is worse than the LCM model you released. Could you give more training details about LCM  distillation ? Such as EMA decay, learning rate, training steps, optimizer. Is there any difference with the implementation of the official LCM (https://github.com/luosiallen/latent-consistency-model.git). Thank you for your help!

**Comment by toshas** [comment]
Hey thanks for your interest, we are preparing an LCM drop (paper, code), follow the repo/us on social media to learn when it is out

**Comment by luochunfen** [comment]
> Hey thanks for your interest, we are preparing an LCM drop (paper, code), follow the repo/us on social media to learn when it is out

OK.  Thank you for your reply.

**Comment by omrastogi** [comment]
Hi, any ideas when you'll be able to release the LCM drop?
